### PR TITLE
[RPA-1792] Fixes bug in schedule process cron select

### PR DESCRIPTION
**Pull request by bhszewczyk**
## Description

Before:
- default schedule value was minute
- once other value that minute was selected other inputs where not mandatory and after submitting the schedule was set to run every minute

After:
- changed default schedule cron value from minute to day
- schedule inputs are now required for all options other than minute

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/components/CustomSelect/CustomSelect.tsx [hunk 1]
```diff
@@ -109,6 +109,8 @@ const CustomSelect: FC<CustomSelectProps> = ({
             renderValue={renderTag}
             autoWidth={false}
             disabled={disabled}
+            required
+            error={!value.length}
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/components/CustomSelect/CustomSelect.tsx:113]: is there appropriate error message?

**Review by star-in-the-jar — APPROVED** [review]
+ version

File: runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/components/CustomSelect/CustomSelect.tsx (hunk 1 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/components/CustomSelect/CustomSelect.tsx:113]: user has to now what did he/she done wrong to make it right. When that's added feel free to merge

Comment (bhszewczyk) [reply runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/components/CustomSelect/CustomSelect.tsx:113]: Consulted this with Ada and agreed that there is no need for this in this particular case - it is handled by browser after submit that this field is required and as long as it it empty the borders are red. If that is truly necessary let me know.

Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-app/components/cron/components/CustomSelect/CustomSelect.tsx:113]: If there are red borders around empty inputs it's correct

**Review by Aeri4a — APPROVED** [review]

**Pull request merged**